packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js

```diff
@@ -0,0 +1,39 @@
+import asyncStates from '@zooniverse/async-states'
+import { flow, getEnv, types } from 'mobx-state-tree'
+
+const FreehandLineReductions = types
+  .model('FreehandLineReductions', {
+    reducer: types.literal('machine-learnt'),
+    error: types.maybeNull(types.frozen({})),
+    loadingState: types.optional(types.enumeration('state', asyncStates.values), asyncStates.initialized),
+    reductions: types.array(types.frozen({})),
+    subjectId: types.string,
+    workflowId: types.string
+  })
+
+  .views(self => {
+    return {
+      consensusLines(frame) {
+        let pathX = []
+        let pathY = []
+
+        self.reductions.forEach(reduction => {
+			    reduction.data.points.forEach(xy => {
+            pathX.push(xy[0])
+            pathY.push(xy[1])
+          })
```

Comment:
```suggestion
			    reduction.data.points.forEach(([x, y]) => {
            pathX.push(x)
            pathY.push(y)
          })
```
Destrutcturing the array might make this easier to read.

Reply:
If there a multiple line reductions, this flattens them into a single line. Is that intentional?

Reply:
This has been fixed so that it can handle any number of reductions.